app/plugin_api.go

```diff
@@ -45,9 +45,14 @@ func (api *PluginAPI) LoadPluginConfiguration(dest interface{}) error {
 	}
 
 	if pluginSettingsJsonBytes, err := json.Marshal(finalConfig); err != nil {
-		return err
+		api.logger.Error("Error marshaling config for plugin", mlog.Err(err))
+		return nil
 	} else {
-		return json.Unmarshal(pluginSettingsJsonBytes, dest)
```

Comment:
Intentional not to return the `err` on `Unmarshal` now?

Reply:
Yes, doesn't work because json.Unmarshal returns an unregistered (with gob) error type.